### PR TITLE
Added CustomGlobalExceptionHandler class, and check valid fields:and …

### PR DESCRIPTION
**Pull request by hanshyn**
…check validation fields: title, author, isbn, price, description, coverImage

### CODE REVIEW

File: src/main/java/hanshyn/onlinebookstore/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+
+    @NotNull
     private String author;
+
+    @NotNull
     private String isbn;
```
Comment (Sanderfuck) [inline src/main/java/hanshyn/onlinebookstore/dto/CreateBookRequestDto.java:22]: Check validation for isbn

File: src/main/java/hanshyn/onlinebookstore/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+
+    @NotNull
     private String author;
```
Comment (Sanderfuck) [inline src/main/java/hanshyn/onlinebookstore/dto/CreateBookRequestDto.java]: Not null but can be empty?

File: src/main/java/hanshyn/onlinebookstore/dto/CreateBookRequestDto.java [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
+
+    @NotNull
     private String isbn;
+
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+
+    @NotNull
     private String description;
```
Comment (Sanderfuck) [inline src/main/java/hanshyn/onlinebookstore/dto/CreateBookRequestDto.java]: You can make limit for description

**Review by Sanderfuck — CHANGES_REQUESTED** [review]

File: src/main/java/hanshyn/onlinebookstore/dto/CreateBookRequestDto.java [hunk 4]
```diff
@@ -1,14 +1,35 @@
...
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.ISBN;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
+    @NotEmpty
```
Comment (okuzan) [inline src/main/java/hanshyn/onlinebookstore/dto/CreateBookRequestDto.java]: for strings let's use \@NotBlank

Comment (okuzan) [reply src/main/java/hanshyn/onlinebookstore/dto/CreateBookRequestDto.java]: NotNull is redundant now

**Review by okuzan — CHANGES_REQUESTED** [review]

**Pull request merged**